src/GitHub.App/ViewModels/LoginTabViewModel.cs

```diff
@@ -18,6 +19,7 @@
 
 namespace GitHub.ViewModels
 {
+    [SuppressMessage("Microsoft.Design", "CA1001:TypesThatOwnDisposableFieldsShouldBeDisposable")]
```

Comment:
Yeah, because it creates a `RecoveryCommand` (which implements `IDisposable`) and assigns it to `NavigateForgotPassword`. I don't think `RecoveryCommand` actually needs to be disposed though - it appears that the only purpose of it is to set its `CanExecute` state.

Should I dispose the `RecoveryCommand` anyway? Or just add a justification?